CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 ## Release Notes
 
+   for on-prem folks who already use LH:
+    - change galley.yaml to FeatureLegalHoldEnabledForWhitelist your teams
+    - that's it!  no other changes needed.  (i think?)
```

Comment:
Could you reword these sentences to be less informal and not use abbreviations? The `FeatureLegalHoldEnabledForWhitelist` should be documented how to use; and release notes should link to that documentation.

Reply:
9af7b37829de11becf25e78203de16e30ac90a55

(Obviously this wasn't ready, i also mentioned yesterday to you that I am going to elaborate on that, but always to go have a reminder.)